source/test/default_mode_test/SymWorld.test.cc

```diff
@@ -1517,99 +1517,208 @@ TEST_CASE("Setup", "[default]") {
+                  REQUIRE(world.GetPop()[i]->HasSym() == true);
+                }
+              }
+            }
+          }
+          WHEN("Free living symbionts are permitted") {
+            config.FREE_LIVING_SYMS(1);
+            WHEN("Starting multiplicity of infection is 1") {
+              config.START_MOI(1);
+              world.Setup();
+              THEN("Free living symbionts fill their world") {
+                for (size_t i = 0; i < world_size; i++) {
+                  REQUIRE(world.GetSymPop()[i] != nullptr);
+                }
+              }
+            }
+            WHEN("Symbionts occupy less than half the spots in the world") {
+              config.START_MOI(0.5);
+              world.Setup();
+              THEN("Symbionts are spaced throughout the world") {
+                for (size_t i = 0; i < world_size - 1; i++) {
+                  if (i % 2 == 1) {
+                    REQUIRE(world.GetSymPop()[i] == nullptr);
+                  }
+                  else {
+                    REQUIRE(world.GetSymPop()[i] != nullptr);
+                  }
+                }
+              }
+            }
+            WHEN("Symbionts occupy more than half the spots in the world") {
+              double smoi = 0.7;
+              config.START_MOI(smoi);
+              world.Setup();
+              THEN("Symbionts are placed sequentially in the world") {
+                for (size_t i = 0; i < world_size * smoi; i++) {
+                  REQUIRE(world.GetSymPop()[i] != nullptr);
+                }
+              }
+            }
+          }
+        } 
+        WHEN("Hosts occupy less than half the spots in the world"){
+          config.POP_SIZE(world_size / 2);
+          world.Setup();
+          THEN("Hosts are spaced throughout the world"){
+            for (size_t i = 0; i < world_size - 1; i++) {
+              REQUIRE(world.IsOccupied(i) == (i % 2 == 0));
+            }
+          }
+        }
+        WHEN("Hosts occupy more than half the spots in the world"){
+          size_t num_hosts = (world_size * 3) / 4;
+          config.POP_SIZE(num_hosts);
+          world.Setup();
+          THEN("Hosts are placed sequentially in the world") {
+            for (size_t i = 0; i < num_hosts - 1; i++) {
+              REQUIRE(world.IsOccupied(i) == true);
+            }
+          }
+        }
+      }
+      WHEN("Grid is off") {
+        double smoi = 0.5;
+        size_t num_hosts = world_size / 2;
+        size_t num_syms = smoi * num_hosts;
+        config.POP_SIZE(num_hosts);
+        config.START_MOI(smoi);
+        world.Setup();
+        THEN("Hosts and symbionts are placed sequentially in the world"){
+          for (size_t i = 0; i < num_hosts - 1; i++) {
+            REQUIRE(world.IsOccupied(i) == true);
+            REQUIRE(world.GetPop()[i]->HasSym() == (i < num_syms));
+          }
+        }
       }
     }
   }
 }
 
-TEST_CASE("SetupSymbionts", "[default]") {
```

Comment:
Probably want to keep the test cases for SetupSymbionts and SetupHosts since we're keeping the methods